doc/core/moving-to-c++.md

```diff
@@ -34,7 +34,19 @@ C++ compatible with
   You can easily see whether PR is free from this issue by checking the result of
   `Artifact Build Linux` test.
 - `thread_local` is not allowed to use on Apple's products because their old OSes
-  (e.g. ios < 9.0) don't support `thread_local`.
+  (e.g. ios < 9.0) don't support `thread_local`. Please use `GPR_TLS_DECL` instead.
+- gRPC main libraries (grpc, grpc+++, and plugins) cannot use following C++ libraries:
+  (Test and example codes are relatively free from this constraints)
+  - `<thread>`. Use `grpc_core::Thread`.
+  - `<condition_variable>`. Use `gpr_cv_init`.
+  - `<future>`
+  - `<mutex>`. Use `gpr_mu_init`.
```

Comment:
Should use grpc_core::Mutex , grpc_core::MutexLock (instead of lock_guard), and grpc_core::ReleasableMutexLock (instead of unique_lock)

Reply:
Done.